runtime/doc/builtin.txt

```diff
@@ -2755,6 +2756,17 @@ feedkeys({string} [, {mode}])				*feedkeys()*
 		Can also be used as a |method|: >
 			GetInput()->feedkeys()
 
+filecopy({from}, {to})					*filecopy()*
+		Copy the file by the name {from} to the name {to}.  The
```

Comment:
just files? or also directories? Ah, I get it from the test, that directory copies are not supported. This should be documented I think

Reply:
Yes.  It copies just files.  I will update the documentation.